es-mode.el

```diff
@@ -438,10 +438,12 @@ in which case it prompts the user."
  completion, if available."
   (interactive (list 'interactive))
   (cl-case command
-    (prefix (let ((sym (company-grab-symbol)))
-              (if (string-match "\"\\(.*\\)\"?" sym)
-                  (match-string 1 sym)
-                sym)))
+    (interactive (company-begin-backend 'company-css))
```

Comment:
Out of curiosity, why does this reference `company-css` which looks like the completion for css-mode?